tests/src/Functional/AlertBannerBlockTest.php

```diff
@@ -32,21 +33,44 @@ class AlertBannerBlockTest extends BrowserTestBase {
   protected $adminUser;
 
   /**
-   * {@inheritdoc}
+   * Place the alert block.
+   *
+   * Utility to place the alert banner block so we can assert
+   * against it's output.
+   *
+   * @param array $settings
+   *   The block settings.
+   *
+   * @return \Drupal\block\Entity\Block
+   *   The placed block.
    */
-  protected function setUp() {
-    parent::setUp();
-
+  protected function placeAlterBlock(array $settings = []) {
```

Comment:
Did you mean placeAlertBlock() or is placeAlterBlock() the intended name?

Reply:
I assume it means `placeAlterBlock` @j4-m as its referenced in a few places as new additions.